qgisprocess_user_2024.Rmd

```diff
@@ -230,6 +230,7 @@ seine_path <- "data/seine.gpkg"
 Run algorithm:
 
 ```{r}
+# remotes::install_github("r-spatial/qgisprocess")
```

Comment:
Well, it's on CRAN so IMO we should prioritize that, i.e. using `install.packages("qgisprocess")`.

I intend to do a CRAN submission one of these days, to make the latest changes available.

Reply:
Agreed!